test/integration/default/verify/controls/aws_ec2_security_groups.rb

```diff
@@ -40,4 +40,11 @@
   describe aws_ec2_security_groups.where(vpc_id: fixtures['ec2_security_group_default_vpc_id']) do
     its('group_ids') { should include fixtures['ec2_security_group_default_group_id'] }
   end
-end
\ No newline at end of file
+end
+
+control "aws_ec2_security_groups" do
+  # Verify you have more than the default security group
+  describe aws_ec2_security_groups do
+    its('count') { should be >= 2 }
```

Comment:
The documented property is `'entries.count'`.  Adding a test for it is a great idea, though!

Reply:
updated this to 'entries.count' as per the docs.